app/BeaconBridge/Controllers/InfoController.cs

```diff
@@ -0,0 +1,42 @@
+using BeaconBridge.Config;
+using BeaconBridge.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+
+namespace BeaconBridge.Controllers;
+
+[ApiController]
+[Route("api/")]
+public class InfoController(IOptions<BeaconInfoOptions> beaconInfoOptions,
+    IOptions<OrganisationOptions> organisationOptions)
+  : ControllerBase
+{
+  private readonly BeaconInfoOptions _beaconInfoOptions = beaconInfoOptions.Value;
+  private readonly OrganisationOptions _organisationOptions = organisationOptions.Value;
+
+  [HttpGet, Route(""), Route("info")]
```

Comment:
what's the purpose of 2 `Route` decorators here?

Reply:
both root and /info are supposed to return the same information, so it makes sense to add both in the decorators instead of using the same code twice

Reply:
Ah I see, cool